public/app/features/alerting/unified/Insights.tsx

```diff
@@ -0,0 +1,179 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import {
+  EmbeddedScene,
+  PanelBuilders,
+  SceneDataTransformer,
+  SceneFlexItem,
+  SceneFlexLayout,
+  SceneQueryRunner,
+  SceneTimeRange,
+} from '@grafana/scenes';
+import { useStyles2 } from '@grafana/ui';
+
+const FIRING_QUERIES_LAST_WEEK = 'sum(count_over_time({from="state-history"} | json | current="Alerting"[1w]))';
+const TOTAL_QUERIES_LAST_WEEK = 'sum(count_over_time({from="state-history"} | json [1w]))';
+
+const WORST_OFFENDERS_ALERTS_THIS_WEEK =
+  'sum by (labels_grafana_folder, group) (count_over_time({from="state-history"} | json | current="Alerting"[1w]))';
+
+//since all cloud instances have a provisioned alert state history data source,
+//we should be able to use its uid here
+const datasourceUid = 'CKA1_2hVz';
+
+function getScene() {
+  const queryRunner1 = new SceneQueryRunner({
+    datasource: {
+      type: 'loki',
+      uid: datasourceUid,
+    },
+    queries: [
+      {
+        refId: 'A',
+        expr: FIRING_QUERIES_LAST_WEEK,
+        title: 'Firing',
+      },
+    ],
+  });
+
+  const queryRunner2 = new SceneQueryRunner({
+    datasource: {
+      type: 'loki',
+      uid: datasourceUid,
+    },
+    queries: [
+      {
+        refId: 'A',
+        expr: TOTAL_QUERIES_LAST_WEEK,
+      },
+    ],
+  });
+
+  const queryRunner3 = new SceneQueryRunner({
+    datasource: {
+      type: 'loki',
+      uid: 'CKA1_2hVz',
+    },
+    queries: [
+      {
+        refId: 'A',
+        instant: true,
+        expr: WORST_OFFENDERS_ALERTS_THIS_WEEK,
+      },
+    ],
+  });
+
+  const queryRunner4 = new SceneQueryRunner({
+    datasource: {
+      type: 'loki',
+      uid: 'CKA1_2hVz',
+    },
+    queries: [
+      {
+        refId: 'A',
+        expr: FIRING_QUERIES_LAST_WEEK,
+      },
+    ],
+    $timeRange: new SceneTimeRange({ from: 'now-1w', to: 'now' }),
```

Comment:
I would like to have a panel that combines the result of two queries that use a different time range to show trending information (for example, number of firing alerts 2 weeks ago compared to firing alerts last week). 
So far I couldn't find a way to do it as the `$timeRange` configuration is set at the `SceneQueryRunner` level, meaning every query performed in that runner uses the same time range, and each panel is rendered using a specific query runner. I'll need to investigate more if this is something that can be done, perhaps using transformations.

Reply:
Unfortunately it doesn't seem to be possible to combine two or more queries with different time ranges within a same panel. As a workaround, we are able to display queries with different time ranges in different panels.